libpod/options.go

```diff
@@ -909,6 +910,20 @@ func WithCommand(command []string) CtrCreateOption {
 	}
 }
 
+// WithRootFS sets the rootfs for the container
+func WithRootFS(rootfs string) CtrCreateOption {
+	return func(ctr *Container) error {
+		if ctr.valid {
+			return ErrCtrFinalized
+		}
+		if _, err := os.Stat(rootfs); err != nil {
+			return errors.Wrapf(err, "error checking path %q", rootfs)
+		}
+		ctr.config.Rootfs = rootfs
```

Comment:
Should check if the container has an image set already, and return an error if it does

Reply:
how?  We cannot set both an image and a rootfs, when `--rootfs` is passed the first argument is a path to the exploded container instead of an image

Reply:
Check that `c.config.RootfsImageID` and `c.config.RootfsImageName` are both "" and return an error if they're not - I want to make sure people don't try and make a container with an image and set rootfs at the same time

Reply:
added